lib/prettyprint.rb

```diff
@@ -42,7 +41,7 @@ class PrettyPrint
   #     output
   #   end
   #
-  def PrettyPrint.format(output=''.dup, maxwidth=79, newline="\n", genspace=lambda {|n| ' ' * n})
+  def PrettyPrint.format(output=''.dup, maxwidth=80, newline="\n", genspace=lambda {|n| ' ' * n})
```

Comment:
It was an off-by-one thing. Previously it was something like `< maxwidth` and now it boils down to `<= maxwidth` but it's a little different because we're subtracting remaining space. It just made it a little nicer to express this way.

Reply:
Ahhh, interesting.  Thanks for the context!